src/types.ts

```diff
@@ -33,46 +33,46 @@ export type Nullable<T> = T | null | undefined
 export type Arrayable<T> = T | Array<T>
 
 export interface SourceMapCompact {
-  file?: string
+  file?: undefined | string
```

Comment:
I prefer moving `undefined` to the end to avoid distractions from less important information.

```suggestion
  file?: string | undefined
```